declarations/Scalingo.history.json

```
@@ -0,0 +1,22 @@
+{
+  "Terms of Service": [
+    {
+      "fetch": "https://scalingo.com/terms-of-service",
+      "select": [
+        ".title",
+        ".mx-sm-auto"
+      ],
+      "validUntil": "2021-08-16T14:36:00.000Z"
```

Isn't this November? 🤔

https://github.com/OpenTermsArchive/contrib-declarations/issues/45

```suggestion
      "validUntil": "2021-11-16T14:36:00.000Z"
```

---

Maybe better to use the date of the last version we got ? https://github.com/OpenTermsArchive/contrib-versions/commit/5fe86743e83b27efbf4783f3e33f2b4e87e4ca83

---

I don't think so: this is the last version, that does not mean this is the [last snapshot](https://github.com/OpenTermsArchive/contrib-snapshots/commits/master/Scalingo/Terms%20of%20Service.html) 🙂 that just means there were no changes recorded in the document since then.

---

We know the URL is incorrect since 16/11. The fact there was no version since 16/08 does not mean we would not be able to regenerate history between 16/08 and 16/11, only that there were no changes recorded.

---

Ok thanks for the clarification, writing history file is really not the easiest part!

---

In fact, the contribution tool takes care of that for you if you use the URL that is indicated within the GitHub issue.

This will also
Fix #47 
Fix #46 
Fix #45